datalad/distribution/get.py

```diff
@@ -237,6 +237,13 @@ def _get_flexible_source_candidates_for_submodule(ds, sm):
                         ds_repo.config[c])
                        for c in ds_repo.config.keys()
                        if c.startswith(candcfg_prefix)]:
+        # ensure that there is only one template of the same name
+        if type(tmpl) == tuple and len(tmpl) > 1:
+            raise ValueError(
+                "There are multiple URL templates for submodule clone "
+                "candidate '{}', but only one is allowed. "
+                "Check your configuration!".format(name)
```

Comment:
```suggestion
                f"candidate '{name}', but only one is allowed. "
                "Check your configuration!"
```
since we are `>= 3.6` in master so we could take advantage from a modern python to make code more readable etc.

also, would it be too much to ask for a test to ensure that issue doesn't resurface and that we don't fall afoul of some variables renaming? ;)

Reply:
Oh, I always thought fstrings where python 3.7 :upside_down_face: - will do!

Reply:
>  would it be too much to ask for a test to ensure that issue doesn't resurface and that we don't fall afoul of some variables renaming? ;)

no, will add one :)

Reply:
test is added.

Reply:
f-string too